tracker/tracker/src/main/app/observer/observer.ts

```diff
@@ -180,16 +187,16 @@ export default abstract class Observer {
     this.app.send(new SetNodeData(id, data));
   }
 
-  private bindNode(node: Node): void {
+  private bindNode(node: Node, isRemove: boolean = false): void {
     const [ id,  isNew ]= this.app.nodes.registerNode(node);
```

Comment:
This is mindcracking.  let's do that on the node unbinding level in the Nodes component - so the node unbinding causes clearance of its subtree?